Add getcellwidths() function (#11837)
vim/vim: runtime/doc/builtin.txt

<		Can also be used as a |method|: >
			GetBufnr()->getbufvar(varname)
<
getcellwidths()						*getcellwidths()*

> 		Returns list for cell widths of character ranges which
> 		overrides by |setcellwidths()|. If cell widths of character
> 		ranges doesn't exist, an empty list is returned.

Reviewer: ```suggestion
		Returns a |List| of cell widths of character ranges overridden
		by |setcellwidths()|.  If no character ranges have their cell
		widths overridden, an empty List is returned.
```

Author: Thank you!
